.eslintrc.json

```diff
@@ -23,6 +23,7 @@
     "import/prefer-default-export": 0,
     "import/no-absolute-path": 0,
     "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/test/*.js"] }],
-    "no-underscore-dangle": 0
+    "no-underscore-dangle": 0,
+    "no-await-in-loop": 0
```

Comment:
This is duplicite key, check line 9. 

```
"no-underscore-dangle": [2, {
      "allow": ["_id"]
}],
```

we don't allow underscores it anywhere in codebase except `_id` and private properties/methods of classes

Reply:
I just added the comma on line 26. I'm gonna remove the key.